src/script/components/panel/UserActions.tsx

```diff
@@ -219,19 +220,24 @@ const UserActions: React.FC<UserActionsProps> = ({
     isNotMe && isAvailable && isNotConnectedUser && canConnect
       ? {
           click: async () => {
-            const connectionIsSent = await actionsViewModel.sendConnectionRequest(user);
-            if (!connectionIsSent) {
+            try {
+              const {connectionStatus, conversationId} = await actionsViewModel.sendConnectionRequest(user);
+
+              // If connection's state is SENT, we create a local 1:1 conversation that will act as a placeholder
+              // before the other user has accepted the request.
+              const connectionConversation =
+                connectionStatus === ConnectionStatus.SENT
+                  ? createPlaceholder1to1Conversation(user, selfUser)
+                  : await actionsViewModel.getConversationById(conversationId);
```

Comment:
This is the main change - we do not create a placeholder for connections that are already accepted (or are in some other state).